source/extensions/filters/http/header_to_metadata/header_to_metadata_filter.cc

```diff
@@ -14,7 +15,7 @@ namespace HttpFilters {
 namespace HeaderToMetadataFilter {
 namespace {
 
-const uint32_t MAX_HEADER_VALUE_LEN = 100;
+const uint32_t MAX_HEADER_VALUE_LEN = 8 * 1024;
```

Comment:
Hmm, shall we make this configurable? Just add a TODO for this PR is fine.

Reply:
Done.